content/darkroom/masking-and-blending/blend-modes.md

```diff
@@ -5,44 +5,159 @@ weight: 20
 
-hue
-: This blend mode mixes hue (color tint) from the input and output images. Lightness and chroma are taken unaltered from the input image.
+subtract
+: Subtract the pixel value of the _output_ from the _input_. When blending in the "RGB (scene)" color space, the pixel values of the output image are multiplied by a value proportional to the "blend fulcrum". Pixel values less than 0 are set to 0.
 
-color
-: This blend mode mixes color (chroma and hue) from the input and output images. Lightness is taken unaltered from the input image. 
+subtract reverse
+: _only available in the "RGB (Scene)" color space_
+: Subtract the pixel value of the _input_ from the _output_. In this case, the "blend fulcrum" parameter of the "RGB (scene)" color space is applied to the input image.
 
-: _Caution: When modules drastically modify hue (e.g. when generating complementary colors) this blend mode can result in strong color noise._
+multiply
+: Multiply the pixel values of the input and output together. As all pixel values are between 0 and 1.0, the resulting output will always be darker. When blending in the "RGB (scene)" color space, this value is further multiplied by a value proportional to the "blend fulcrum".
+
+multiply reverse
+: _only available in the "RGB (Scene)" color space_
+: This mode works in the same way as the "multiply" mode except that in the `final_output` equation given at the top of this page, the `input` parameter is replaced with `module_output` (the output of the module before blending is applied).
+
+divide
+: Divide the pixel values of the input by the output. When blending in the "RGB (scene)" color space, the pixel values of the output image are multiplied by a value proportional to the "blend fulcrum".
+
+divide reverse
+: _only available in the "RGB (Scene)" color space_
+: Divide the pixel values of the _output_ by the _input_. In this case, the "blend fulcrum" parameter of the "RGB (scene)" color space is applied to the input image.
+
+screen
+: _not available in the "RGB (Scene)" color space_
+: Invert the input and output pixel values, multiply those values together and invert the result. This yields the opposite effect to "multiply" mode -- the resulting image is usually brighter, and sometimes “washed out” in appearance.
+
+average
+: Return the average of the input and output pixel values.
+
+difference
+: Return the absolute difference between the input and output pixel values.
+
+geometric mean
+: Return the square root of the product of the input and output pixel values.
+
+harmonic mean
+: Return the product of the input and output pixel values, multiplied by 2 and divided by their sum.
+
+# contrast enhancing modes
+
+The following modes are not available in the "RGB (Scene)" blending color space as they rely on an assumption of "50% mid grey" which only applies to display-referred color spaces.
```

Comment:
display-referred AND non-linear, because display-referred only means the signal is bounded in [0; 100] % but says nothing about the encoding.

Reply:
can you clarify what you'd like me to change here please.

Reply:
> Edit: or should it just be "...only applies to display-referred and non-linear color spaces."

yes